rust/src/libnm_dbus/connection/vlan.rs

```diff
@@ -0,0 +1,52 @@
+use crate::dbus_value::{own_value_to_string, own_value_to_u32};
+use crate::NmError;
+use std::collections::HashMap;
+use std::convert::TryFrom;
+
+#[derive(Debug, Clone, PartialEq, Default)]
+pub struct NmSettingVlan {
+    pub base: Option<String>,
```

Comment:
Please use `parent` to be consistent with NM dbus interface.

Reply:
Done